semgrep-core/src/matching/SubAST_generic.ml

```diff
@@ -179,6 +179,7 @@ let subexprs_of_expr with_symbolic_propagation e =
           xml_body
   (* currently skipped over but could recurse *)
   | Constructor _
+  | RegexpTemplate _
```

Comment:
Seems like something we should recurse on.
We do for Interpolated strings I think (represented by Calls and IdSpecial).

Reply:
yes. I added this with a test.